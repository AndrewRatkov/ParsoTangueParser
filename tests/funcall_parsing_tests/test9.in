h("cucu()" * f(30) == g(), "meow:)")