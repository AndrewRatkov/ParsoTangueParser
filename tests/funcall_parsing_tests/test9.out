[VOID h]------->[INT,==]------->[STR,* ]------->[STR,"cucu()"]
    |               |               |
    |               |               +---------->[INT f]------->[INT,30]
    |               |
    |               +---------->[STR g]
    |
    +---------->[STR,"meow:)"]
